cypress/e2e/failedLogin.cy.js

```diff
@@ -0,0 +1,17 @@
+describe("failed login function", () => {
+  it("visits page and fails logging in", () => {
+    cy.visit("https://mar-ols.github.io/social-media-client/");
```

Comment:
```suggestion
    cy.visit("/");
```

You should be targeting the code in your branch, not the code that has been deployed and merged already.